eureka-core/build.gradle

```diff
@@ -10,7 +10,8 @@ dependencies {
     api "com.amazonaws:aws-java-sdk-route53:${awsVersion}"
     api "jakarta.servlet:jakarta.servlet-api:${servletVersion}"
     api 'jakarta.inject:jakarta.inject-api:2.0.1'
-    api 'com.thoughtworks.xstream:xstream:1.4.19'
+    api 'com.thoughtworks.xstream:xstream:1.4.20'
+    implementation 'com.google.guava:guava:33.0.0-jre'
```

Comment:
Since this PR is about servo->spectator I'm leaving the guava dependency in place (but updating to the latest version), I can follow up with another PR that will replace the guava usages